libs/ui/src/components/atoms/input/input.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Input from './input';
+
+export default {
+  title: 'Atoms/Input',
+  component: Input,
+} as ComponentMeta<typeof Input>;
+
+export const InputAtom: ComponentStory<typeof Input> = (args) => <Input />;
+export const InputAtom2: ComponentStory<typeof Input> = (args) => <Input />;
+export const InputAtom3: ComponentStory<typeof Input> = (args) => <Input />;
```

Comment:
Is the return type required here? In general I am a fan of letting typescript inferences for typescript